src/components/Differential/Differential.jsx

```diff
@@ -991,7 +991,7 @@ class Differential extends Component {
         if (multifeaturePlot.length === 1) {
           try {
             const isMultiModelMultiTestVar = isMultiModelMultiTest(
-              plot.plotType,
+              multifeaturePlot[0].plotType,
```

Comment:
Previously we only had studies with several multi-feature plots.  In the problematic case, there was a single multi-feature plot, and the code that handled it (just this line) was wrong.